### PR TITLE
docs: lots of changes

### PR DESCRIPTION
**Pull request by VaibhavSys**

### CODE REVIEW

**Review by orxngc — DISMISSED** [review]

**Comment by wdhdev** [comment]
What is this PR achieving? All I see are changes being reverted?

**Comment by VaibhavSys** [comment]
> What is this PR achieving? All I see are changes being reverted?

Mainly:

- Adding resources page
- Changing separator in paths from _ to -
- Adding FAQ page
- Moving some guides (deleting domain, editing file) to FAQ
- Fixing link warnings & adding documentation edit option
- Some other minor things

**Comment by wdhdev** [comment]
Are you able to alphabetise everything (like in the sidebar)

File: src/docs/discord-verification/index.md [hunk 1]
```diff
@@ -3,45 +3,39 @@
...
 {
     "owner": {
-        "username": "github-username",
-        "email": "email@address"
+        "username": "<github-username>",
+        "email": "<email@address>"
     },
     "record": {
-        "TXT": "discord-verification-string"
+        "TXT": "<discord-verification-string>"
```
Comment (wdhdev) [inline src/docs/discord-verification/index.md:34]: Please revert this.

File: src/mkdocs.yml [hunk 2]
```diff
@@ -23,13 +27,13 @@ markdown_extensions:
...
-  - Edit Your Domain: edit_domain/index.md
-  - GitHub Pages: github_pages/index.md
-  - GitHub Pages Domain Verification: github_pages_verification/index.md
+  - Domain Structure: domain-structure/index.md
+  - FAQ: faq/index.md
+  - Resources: resources/index.md
+  - Cloudflare Pages: cloudflare-pages/index.md
+  - Discord Domain Verification: discord-verification/index.md
+  - GitHub Pages: github-pages/index.md
+  - GitHub Pages Domain Verification: github-pages-verification/index.md
```
Comment (wdhdev) [inline src/mkdocs.yml:36]: Alphabetise this please.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: src/docs/discord-verification/index.md (hunk 1 above)
Comment (VaibhavSys) [reply src/docs/discord-verification/index.md]: Done

File: src/mkdocs.yml (hunk 2 above)
Comment (VaibhavSys) [reply src/mkdocs.yml:36]: **NO.**

**Pull request merged**